addon/modifiers/did-insert.js

```diff
@@ -22,19 +22,20 @@ import Ember from 'ember';
   ```
 
   By default, the executed function will be unbound. If you would like to access
-  the component context in your function, use the `action` helper as follows:
+  the component context in your function, use the `action` decorator as follows:
 
   ```handlebars
-  <div {{did-insert (action this.incrementCount)}}>first</div>
-  <div {{did-insert (action this.incrementCount)}}>second</div>
+  <div {{did-insert this.incrementCount}}>first</div>
+  <div {{did-insert this.incrementCount}}>second</div>
 
   <p>{{this.count}} elements were rendered</p>
   ```
 
-    ```js
+  ```js
   export default Component.extend({
     @tracked count = 0;
 
+    @action
```

Comment:
Can't use decorators in `.extend()` here, you'd have to use:

```js
export default Component.extend({
  count: tracked({ value: 0 }),

  incrementCount: action(function() {
    this.count++;
  })
})
```

Reply:
Sorry, yes - I, missed that. I'll update now, thanks